dashboard/app/views/layouts/_student_information_interstitial.html.haml

```diff
@@ -0,0 +1,47 @@
+.modal#student-information-modal{'data-backdrop' => 'static', 'data-keyboard' => 'false', display: 'none'}
+  .modal-dialog
+    .modal-content.no-modal-icon
+      %h1{style: 'margin: 10px 0;font-size: 32px'}= t('activerecord.attributes.user.finish_sign_up_header')
+      %p{style: 'font-size: 13px; line-height: 18px; color: #333'}
+        = t('parent_mailer.parent_email_added_subject')
+        = t('activerecord.attributes.user.finish_sign_up_header_account_information')
+        = t('age_interstitial.disclaimer')
+      = form_for(current_user, url: registration_url('user'), html: {id: 'edit_user'}) do |f|
+        .form-group
+          = f.label :age, t('signup_form.age')
+          = f.select :age, User::AGE_DROPDOWN_OPTIONS, include_blank: t('signup_form.select_age')
+        - if Policies::Lti.lti?(current_user)
+          .form-group
+            = f.label :us_state, t('activerecord.attributes.user.us_state')
+            = f.select :us_state, User::US_STATE_DROPDOWN_OPTIONS, include_blank: t('school_info.select_state')
+          .field
+            = f.label :gender_student_input, t('gender')
+            = f.text_field :gender_student_input, placeholder: t('type_here')
```

Comment:
In other areas where we present a gender input, we don't provide any placeholder text. We received feedback during internal testing that the tone is too imperative for an optional field. Can you remove the placeholder text?

Reply:
Will do!